en-US/Punctuation.xml

```diff
@@ -185,32 +223,43 @@
 
 			</listitem>
 
-		</itemizedlist>
-		 <para>
-			Use a semicolon to separate items in a series if the items contain commas.
+		</itemizedlist>		
+
+		<para>
+			In the following example, a colon is used for a step that introduces or leads into a command:	
```

Comment:
For simplicity, could we remove the "or" from this sentence? "In the following example, a colon is used for a step that introduces a command:"